WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/WebView.swift

```diff
@@ -14,6 +14,9 @@ struct WebView: UIViewRepresentable {
         }
     }
 
+    private let webView = WKWebView()
+    private let progressView = WebProgressView()
+
     let url: URL
```

Comment:
Please also keep all the other properties private.

Reply:
I added them in https://github.com/woocommerce/woocommerce-ios/pull/11501/commits/6859323031d2cc42809b192a2dfc4661c5004292

I left `exitTrigger` and `urlToTriggerExit` alone, though. From what I can find they're not used elsewhere in the code, but they're not part of the `init` either, so setting them private will make them unsettable.

I think it's just an effect of `WebView` being a mirror of `AuthenticatedWebView` which has and uses them. Wdyt we should do?

Reply:
We should add parameters with default values for those while keeping them private if we want to mirror `AuthenticatedWebView`. I tried setting them to private, and the project built just fine, meaning we don't use the properties anywhere. You can also consider removing them as they're unused.

Reply:
Yeah I think removing will be most straightforward, thanks for your suggestion.

Reply:
https://github.com/woocommerce/woocommerce-ios/pull/11501/commits/c0cb49842196fa7b28e36a2d7577f405ef316d72